networkit/graphtools.pyx

```diff
@@ -462,15 +462,12 @@ cdef class GraphTools:
 	def subgraphFromNodes(Graph graph, vector[node] nodes, includeOutNeighbors=False, includeInNeighbors=False, bool_t compact = False):
 		"""
 		subgraphFromNodes(graph, list(int) nodes, includeOutNeighbors=False, includeInNeighbors=False, compact = False)
-		
-		Returns an induced subgraph of this graph (including potential edge
-		weights/directions)
```

Comment:
Could you also move the section below ("The subgraph contains...") in the Return docstrings?